README.md

```diff
@@ -1,33 +1,25 @@
-# openweathermap-api
+# newyorktimes-api
+https://developer.nytimes.com
 
 ### Description
-This application fetches two types of data from the openweathermap API; the current weather data and the 5 day forecast. Using spec, there are sets of unit tests which test the validity of the openweathermaps data. Numerous unit tests have been set-up and tested using the rspec gem. The use of HTTParty is required to fetch the API, allowing it to be tested.
+This application uses the newyorktimes API to query a searched for movie. Through the service object model, testing of the returned data was performed to check for validity and consistency. This was tackled by using RSpec which is a Ruby testing tool. In addition, HTTParty is required to fetch and interact with the API. 
```

Comment:
This is not an application. We have created a SOM. What did you mean by application